rdp.c

```diff
@@ -98,38 +129,35 @@ rdp_recv(uint8 * type)
 	static STREAM rdp_s;
 	uint16 length, pdu_type;
```

Comment:
> rdp.c:130:17: warning: unused variable ‘pdu_type’ [-Wunused-variable]

You removed all use of `pdu_type` from `rdp_recv`, so we can remove the local variable as well.